### PR TITLE
Refactor/destructure styles

### PR DESCRIPTION
**Pull request by CFDevelopment**
- Refactoring code
- removing unused imports
- restructuring style objects
- ES lint warning being removed

### CODE REVIEW

File: src/reducers/contacts/contactReducer.js [hunk 1]
```diff
@@ -4,50 +4,48 @@ const INITIAL_STATE = {
...
+ * Reducer used to handle all actions occurring during the process of
  * adding a new contact. Each case represents an Action
  */
 export default (state = INITIAL_STATE, action) => {
   switch (action.type) {
     case actions.ADDING_CONTACT:
       var contact = state.currentContact;
 
-      //Object.keys(contact)[0]
+      // Object.keys(contact)[0]
```
Comment (priom) [inline src/reducers/contacts/contactReducer.js]: @CFDevelopment r these unused code?

File: src/reducers/contacts/contactReducer.js [hunk 2]
```diff
@@ -4,50 +4,48 @@ const INITIAL_STATE = {
...
-      return { ...state, currentContact: contact }
+      return { ...state, currentContact: contact };
+
 
-    
     case actions.COMPLETE_CONTACT:
-      var old = state.contacts
+      var old = state.contacts;
 
       // old.push(state.currentContact)
```
Comment (priom) [inline src/reducers/contacts/contactReducer.js]: @CFDevelopment more here

File: src/reducers/contacts/contactReducer.js [hunk 3]
```diff
@@ -4,50 +4,48 @@ const INITIAL_STATE = {
...
 
       if (actionKey in contact) {
-        contact[actionKey] = actionKeyValue
+        contact[actionKey] = actionKeyValue;
       } else {
-        contact[actionKey] = actionKeyValue
+        contact[actionKey] = actionKeyValue;
       }
 
       // console.log("Contact Details: ");
```
Comment (priom) [inline src/reducers/contacts/contactReducer.js]: @CFDevelopment and also here....

File: src/reducers/wallet/NewWalletSetup.js [hunk 4]
```diff
@@ -7,47 +7,41 @@ const INITIAL_STATE = {
...
 /**
  * Reducer used to handle all actions occuring during the process of creating a wallet.
- * 
+ *
  * Also handles the action invoked by using the QrScanner Component
  */
 
 export default (state = INITIAL_STATE, action) => {
-  //console.log(action);
+  // console.log(action);
```
Comment (priom) [inline src/reducers/wallet/NewWalletSetup.js]: clean all console.log completely

File: src/reducers/contacts/contactReducer.js (hunk 1 above)
Comment (CFDevelopment) [reply src/reducers/contacts/contactReducer.js]: comments mean unused yes, I will verify and remove them

File: src/reducers/wallet/NewWalletSetup.js [hunk 5]
```diff
@@ -7,47 +7,41 @@ const INITIAL_STATE = {
...
-        //add the selected coin to the token list
+        // add the selected coin to the token list
         newTokens = [...current, action.payload];
-
       } else {
-        //Delete the selected coin from the token list
-        newTokens = [...current.slice(0, index), ...current.slice(index + 1)]
+        // Delete the selected coin from the token list
+        newTokens = [...current.slice(0, index), ...current.slice(index + 1)];
         // current
```
Comment (priom) [inline src/reducers/wallet/NewWalletSetup.js]: what's this for?

Comment (CFDevelopment) [reply src/reducers/wallet/NewWalletSetup.js]: fixed

File: src/reducers/wallet/NewWalletSetup.js (hunk 4 above)
Comment (CFDevelopment) [reply src/reducers/wallet/NewWalletSetup.js]: fixed

File: src/reducers/contacts/contactReducer.js (hunk 3 above)
Comment (CFDevelopment) [reply src/reducers/contacts/contactReducer.js]: fixed

File: src/reducers/contacts/contactReducer.js (hunk 2 above)
Comment (CFDevelopment) [reply src/reducers/contacts/contactReducer.js]: fixed

File: src/reducers/contacts/contactReducer.js (hunk 1 above)
Comment (CFDevelopment) [reply src/reducers/contacts/contactReducer.js]: fixed

File: src/reducers/wallet/NewWalletSetup.js [hunk 6]
```diff
@@ -21,34 +21,22 @@ const INITIAL_STATE = {
...
       return { ...state, QrData: action.payload };
-    case actions.CREATING_NEW_WALLET:
-      console.log('wallet reducer: ');
-      console.log(action.payload);
+    case actions.CREATING_NEW_WALLET:      
       return { ...state, newWallet: true, wallet: action.payload };
     case actions.NEW_WALLET_NAME:
       return { ...state, walletName: action.payload };
     case actions.ADD_TOKEN_SETUP:
       var current = state.tokens;
```
Comment (priom) [inline src/reducers/wallet/NewWalletSetup.js]: use let or const instead of var @CFDevelopment

Comment (CFDevelopment) [reply src/reducers/wallet/NewWalletSetup.js]: u need to let up lol

Comment (CFDevelopment) [reply src/reducers/wallet/NewWalletSetup.js]: fixed

**Pull request merged**